pkg/cluster/storage.go

```diff
@@ -30,12 +47,17 @@ func (c *Cluster) save(ctx context.Context) error {
 		return err
 	}
 
-	if err := storageClient.Create(ctx, storageKey(c.config.Token), data); err != nil {
+	_, _, err = c.getBootstrapKeyFromStorage(ctx, storageClient, normalizedToken)
+	if err != nil {
+		return err
+	}
+
+	if err := storageClient.Create(ctx, storageKey(normalizedToken), data); err != nil {
 		if err.Error() == "key exists" {
-			logrus.Warnln("Bootstrap key exists. Please follow documentation updating a node after restore.")
+			logrus.Warnln("bootstrap key exists please follow documentation updating a node after restore")
```

Comment:
This isn't quite valid grammar; if you're going to get rid of the period then replace it with a comma or semicolon
```suggestion
			logrus.Warnln("bootstrap key exists; please follow documentation updating a node after restore")
```